views/apartments/id_tenants.erb

```diff
@@ -0,0 +1,6 @@
+<% Apartment.find(@id).tenants.each do |tenant| %>
```

Comment:
instead, create an instance variable in your controller. This will help separate your concerns